### DISCUSSION TITLE
farming on multiple harvesters help

### DISCUSSION THREAD

**Discussion by terryxsq** [Chia-Network/chia-blockchain#5946] [Support]
I have been following the guide to setup harvesters and main machine.

i got to the step to test and set log to INFO. But i could not see any of the below logs in debug.log
```
[time stamp] farmer farmer_server   : INFO   -> new_signage_point_harvester to peer [harvester IP address] [peer id - 64 char hexadecimal]
[time stamp] farmer farmer_server   : INFO   <- farming_info from peer [peer id - 64 char hexadecimal] [harvester IP address]
[time stamp] farmer farmer_server   : INFO   <- new_proof_of_space from peer [peer id - 64 char hexadecimal] [harvester IP address]
```
Has the logging changed and the wiki is outdated?

**Comment by 21497724** [comment]
Have you set our log level to DEBUG?

**Comment by BrandtH22** [comment]
@terryxsq , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
